tests/index.ts

```diff
@@ -0,0 +1,24 @@
+import {suite} from 'uvu'
+import {is} from 'uvu/assert'
```

Comment:
Fix these ESLint warnings.

Reply:
Solved in [`a0e51fe`](https://github.com/LeNitrous/azur-lane-api/pull/10/commits/a0e51feaa4e39b9353674d5ebe347388f657446d)